Master - FAQ - Selenium updated test - Agent part2

s7design opened the pull request:
Hi @carlosfrodriguez 

Here are the rest updated Selenium test from Agent directory. Whit this PR, all Selenium test from FAQ are updated.

Regards
Zoran

carlosfrodriguez commented:
Hi @s7design,

Thank you for your pull request.

I got some errors after merging, can you please take a look on it?

```
  /Users/Carlos/Dev/OTRS/Public/otrspm-5_0/scripts/test/Selenium/Agent/AgentFAQPrint.t #45 - Exception in Selenium': OTRS API verification failed after page load. at /Users/Carlos/Dev/OTRS/Public/otrspm-5_0/Kernel/System/UnitTest/Selenium.pm line 224.

  /Users/Carlos/Dev/OTRS/Public/otrspm-5_0/scripts/test/Selenium/Agent/AgentFAQSearch.t #37 - Exception in Selenium': An element could not be located on the page using the given search parameters: #SearchProfile,css selector at (eval 192) line 107.
```

((enjoy))
cr

s7design commented:
Hi Carlos,

Thanks for the follow-up. I tested it again, and there could be problem with waiting for loading page in AgentFAQSearch, however on my system it works fine. It is a little bit strange, because of that we are switching to Verified methods. I added one WaitFor on the place which made a problem. To be honest I prefer option without that WaitFor, but I don't know why it did not work on your system.

In the second test  AgentFAQPrint, it was my mistake I switched to VerifiedGet but there is loading PDF view for print screen and there is not possible to call OTRS API, I rolled back on simple get() method  of the Selenium web driver.

Now, these two test works fine for me, please let me know if you have any questions.

Regards
Zoran

carlosfrodriguez commented:
Hi @s7design,

I was able to fix the Search test, apparently there was a missing WaitFor the fist time the test changes the search options, I re-added and now it works.

The pull request was merged in https://github.com/OTRS/FAQ/commit/2efe1a084d1f080f3b32cf57d296df377bda8b45.

Thank you!


((enjoy))
cr
